src/domain/aboutus/components/who_section/WhoSection.module.scss

```diff
@@ -1,78 +1,70 @@
 @import '../../../../utils/variables';
+@import '../../aboutUsMixin';
 
 .whoSection {
-    position: relative;
-    top: -150px;
-    padding: 0% 15%;
-    height: fit-content;
-    display: flex;
-    box-sizing: border-box;
-    padding-bottom: 50px;
-}
+    @include aboutUs('section');
+    background-color: $bgColor3;
 
-.textSection {
-    width: 50%;
-}
+    .content {
+        @include aboutUs('content');
+        max-width: 1320px;
```

Comment:
I think the who section and why section would work better if they were the same width - so either make the why section 1320px or the who section 1240!

Reply:
Yes, these can change. I was not sure how by the pictures you wanted these to be.

Reply:
Not something for this pr, but idk if we can make this something that is consistent across the site? If it something that is going to be reused across pages it would be nice to have a global sass variable for content widths at different screen sizes but then I don't know if that actually something that fits with the designs?